content/doc/book/system-administration/admin-password-reset.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+==Admin-Password-Reset Jenkins
+As a system administrator, you may need to reset the Jenkins admin password if you have lost or forgotten it. To cover this possibility we have steps to follow to reset the admin password.
```

Comment:
```suggestion
As a system administrator, you may need to reset the Jenkins admin password if you have lost or forgotten it.
To cover this possibility we have steps to follow to reset the admin password.
```
suggestion for single sentence per line formatting, so that reviews and feedback can be provided easier.